clang/test/SemaSYCL/warn-constexpr-kernel-arg.cpp

```diff
@@ -0,0 +1,38 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -internal-isystem %S/Inputs -fsyntax-only -Wsycl-strict -sycl-std=2020 -verify %s
```

Comment:
Is `-Wsycl-strict` flag necessary here?
```suggestion
// RUN: %clang_cc1 -fsycl -fsycl-is-device -internal-isystem %S/Inputs -fsyntax-only -sycl-std=2020 -verify %s
```

Reply:
It is not required for this test. I will remove it.